Gemfile

```diff
@@ -9,3 +9,10 @@ unless RUBY_PLATFORM
   gem 'pry-stack_explorer'
   gem 'pry-rescue'
 end
+
+# Gem a dependency of Capybara, new version of rack 2.0 onwards require ruby 2.2 and above.
+if RUBY_VERSION < "2.2.2"
```

Comment:
I do not think this will be an issue as examples below show

```
irb(main):001:0> "2.20.0" < "2.2.2"
=> false
```

```
irb(main):002:0> "2.19.0" < "2.2.2"
=> true
```